src/test/java/com/amazonaws/services/dynamodbv2/GetLocksByPartitionKeyTest.java

```diff
@@ -0,0 +1,175 @@
+package com.amazonaws.services.dynamodbv2;
+
+import com.amazonaws.services.dynamodbv2.model.LockNotGrantedException;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.Test;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class GetLocksByPartitionKeyTest extends InMemoryLockClientTester {
```

Comment:
Please add javadocs summarizing purpose of test classes added in this PR

Reply:
Added.